gordo_components/dataset/sensor_tag.py

```diff
@@ -33,10 +33,11 @@
 ]
 
 
-def _asset_from_tag_name(tag_name: str):
+def _asset_from_tag_name(tag_name: str, default_asset: str = None):
     """
-    Resolves a tag to the asset it belongs to, if possible.
-    Returns None if it does not match any of the tag-regexps we know.
+    Resolves a tag to the asset it belongs to, if possible. Raises
+    SensorTagNormalizationError if it does not match any of the tag-regexps we know
+    and default_asset is not set.
```

Comment:
Should this just follow the standard numpy docs if we're also adding what it can raise?